### PR TITLE
Event details

### PR DESCRIPTION
**Pull request by lucasgustaf**
Added comment and group join functionality

### CODE REVIEW

File: src/reducers/events.js [hunk 1]
```diff
@@ -4,15 +4,101 @@ const events_test_data = [{name: "Soccer game", location: "Empire Field, Vancouv
                           {name: "MTG Tournament", location: "Vancouver Convention Centre", date: new Date("2020-01-04"), genre: ['games', 'card games', 'tournaments', 'magic'], creator: {name: "Joan Smith"}, participants: [{name: "Daniel Boyce"}, {name: "Carol Wisnewski"}, {name: "Samuel Zubrus"}, {name: "Gary Lee"}, {name: "Andrew Gergich"}, {name: "Jennifer Quincey"}, {name: "Joel Quincey"}, {name: "Joel Harland"}, {name: "Carla Gregg"}], description: "Magic The Gathering tournament. Winner takes home $25,000 grand prize", comments: []}];
 
 
+
+
+const addParticipant = (user, event, events) => {
+        let j = -1;
+        for (let i = 0; i < events.length; i++) {
+            if (JSON.stringify(events[i]) === JSON.stringify(event)) {
```
Comment (tsuiswz) [inline src/reducers/events.js:12]: Do you guys want to use lodash for object equality checks instead? JSON.stringify accounts for order, so 2 objects with same key value pairs will be different if the order is different.

{"firstName": "hi", "lastName": "ho"} != {"lastName": "ho", "firstName": "hi"} with stringify.

File: src/reducers/events.js [hunk 2]
```diff
@@ -4,15 +4,101 @@ const events_test_data = [{name: "Soccer game", location: "Empire Field, Vancouv
...
+                j = i;
+            }
+        }
+        let containsUser = false;
+        for (let i = 0; i < event.participants.length; i++) {
+            if (JSON.stringify(event.participants[i]) === JSON.stringify(user)) {
+                containsUser = true;
+            }
+        }
+        if (!containsUser) {
```
Comment (tsuiswz) [inline src/reducers/events.js:22]: Same as above for finding user existence. I think findIndex() or participants.include(user) should be better alternatives.

Comment (tsuiswz) [reply src/reducers/events.js:22]: I think you also have the same scenario for some of the code below. I won't leave a comment but if you could fix those as well thank you.

**Review by tsuiswz — CHANGES_REQUESTED** [review]

File: src/reducers/events.js [hunk 3]
```diff
@@ -1,18 +1,82 @@
...
 
 
+
+
+const addParticipant = (user, event, events) => {
+        let eventIndex = events.findIndex((element) => {return _.isEqual(element,event)});
+        if (!JSON.stringify(event.participants).includes(JSON.stringify(user))) {
+            event.participants = event.participants.concat(user);
+        }
+        if (eventIndex+1) {
```
Comment (tsuiswz) [inline src/reducers/events.js:15]: Is this suppose to be plus 1?

Comment (lucasgustaf) [reply src/reducers/events.js:15]: Yes I think so. The first Index would be 0 which would evaluate to false when we want it to be true no?

**Pull request merged**